pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -76,7 +76,7 @@ func (l *LibvirtDomainManager) prepareMigrationTarget(
 		return fmt.Errorf("conversion failed: %v", err)
 	}
 
-	dom, err := l.preStartHook(vmi, domain, true)
+	dom, err := l.preStartHook(vmi, domain, true, options)
```

Comment:
Same question: Why do we need this for the migration scenario when the domain definition is expected to arrive from the source.

Reply:
same answer:)